lib/parse/data/parseRssItems.ts

```diff
@@ -84,6 +84,30 @@ const parseRssItems = (
           // episode was filtered out. Prepend it to items.
           return [episode, ...items];
         },
+        // Sort Serial feed items.
+        (items: IRssItem[]) =>
+          rssItunesType === 'serial'
+            ? [...items].sort((a, b) => {
```

Comment:
These ternary's with large blocks of code inside of them are a bit difficult to read.  But maybe I'm just not used to seeing them.

Reply:
This sort started out fairly short, but probably should have been refactored into a named function to make this ternary more readable. We can refactor this now or wait for the addition of additional sorting steps.

Reply:
Either way is fine by me.